gloo/benchmark/runner.cc

```diff
@@ -249,7 +249,8 @@ void Runner::run(BenchmarkFn<T>& fn, size_t n) {
     // Create warmup jobs for every thread
     std::vector<std::unique_ptr<RunnerJob>> jobs;
     for (auto i = 0; i < options_.threads; i++) {
-      auto fn = [&benchmark = benchmarks[i]] { benchmark->run(); };
+      auto& benchmark = benchmarks[i];
+      auto fn = [&benchmark] { benchmark->run(); };
```

Comment:
does this make any differences on performance? Or just better readability?

Reply:
No, it's to make the compiler stop emitting a warning that this is a C++14 extension...